doc/Makefile

```diff
@@ -21,9 +21,10 @@ default:
 all: aspect.tag manual.pdf
 
 # generate .prm.out from .prm using the annotate.pl script:
-%.prm.out: %.prm
+%.prm.out: %.prm Makefile
 	@echo "generating '$@' from '$<'..."
 	@perl manual/cookbooks/annotate.pl $< >$@
+	@for i in `seq 1 10`; do sed -i 's/{\([^!]*\)!\([^!]*\)!\([^!]*\)!\([^}]*\)}/{\1!\2!\3\/\4}/' $@; done
```

Comment:
You don't use `$i` in the rule. Is it correct that all you want to do is just keep replacing the fourth exclamation point by a slash until you've really found all, and expect that trying ten times is enough? Maybe add a comment (and also add a comment that explains why you do it at all, just in case we ever forget about the limit of 3).